dds_glossary/services.py

```diff
@@ -93,6 +96,61 @@ def __init__(
         self.data_dir.mkdir(parents=True, exist_ok=True)
         onto_path.append(str(self.data_dir))
 
+    @staticmethod
+    def check_concept_scheme_exists(
+        concept_scheme: ConceptScheme | None,
+        concept_scheme_iri: str,
+    ) -> ConceptScheme:
+        """
```

Comment:
If this is meant to be a utility to raise if not found, then I'd rename and go with a similar pattern as the requests library, and call this [`raise_for_status`[1]](https://3.python-requests.org/_modules/requests/models/#Response.raise_for_status) or similar; I'd also accept `Any` since we don't really care about the object type and then this can be reused elsewhere. But why can't we return `-> None`? I'm not sure there's a huge benefit in extracting this away though, and it would probably be much easier to use [`one()`[2]](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.one) instead of `one_or_none()` directly at the database level; is there any situation so far where we actually want to return `None` back to the user?


[1] - https://3.python-requests.org/_modules/requests/models/#Response.raise_for_status
[2] - https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.one

Reply:
I created it at first to accept `Base` by `mypy` was very angry about it. I think we could probably just by now go with `one()`. I was trying to keep the database functions generic and keep error handling in `GlossaryController` but seems not to be needed. I will adjust accordingly.

Reply:
@jsvgoncalves resolved in e137de70832c890de8938f652adb12518863378e